### PR TITLE
Updates form UI to display theme selection

### PR DESCRIPTION
**Pull request by trevormast**
Removes css pseudo selector logic for image highlighting and uses Jquery to modify css upon click.

### CODE REVIEW

**Comment by trevormast** [comment]
addresses issue #75

File: views/new.slim [hunk 1]
```diff
@@ -30,41 +30,46 @@ section#wrapper
 
           input type="submit" value="Create"
 
-          a class="button" href="#" data-featherlight="#modal-box" style="float: right" Advanced Options
+          a class="button" href="#" data-featherlight="#modal-box" style="float: right" Theme Options
           
-      div
+      div id="side-form"
```
Comment (amoose) [inline views/new.slim:35]: use slim shorthand: `#side-form`

File: views/new.slim [hunk 2]
```diff
@@ -30,41 +30,46 @@ section#wrapper
...
         a href="http://github.com/#{@user.login}" target="_blank" style="border-bottom: 0 none"
           img src="#{@user.avatar_url}" style="width: 100px; border-radius: 50%;"
         h2 Submit your Order
         p Enter some information about your new blog to get started
+        div id="theme-box"
```
Comment (amoose) [inline views/new.slim:40]: slim shorthand `#theme-box`

File: views/new.slim [hunk 3]
```diff
@@ -30,41 +30,46 @@ section#wrapper
...
+          h2 Theme
+          img id="current-theme" src="#{root_url}/images/clean-jekyll.png"
         br clear="both" 
 
+        
+
       div class="featherlight" id="modal-box"
```
Comment (amoose) [inline views/new.slim:47]: `#modal-box.featherlight`

**Pull request merged**